src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEModuleSymbol.cs

```diff
@@ -705,12 +705,13 @@ internal override bool UtilizesNullableReferenceTypes
             }
         }
 
-        internal override bool NullableOptOut
+        internal override bool NonNullTypes
         {
             get
             {
-                bool optOut;
-                return _module.HasNullableOptOutAttribute(EntityHandle.ModuleDefinition, out optOut) ? optOut : base.NullableOptOut;
+                // The default for PE modules is [NonNullTypes(false)]
+                bool nonNullTypes;
+                return _module.HasNonNullTypesAttribute(EntityHandle.ModuleDefinition, out nonNullTypes) ? nonNullTypes : base.NonNullTypes;
```

Comment:
>: base.NonNullTypes [](start = 120, length = 19)

Isn't `PEModuleSymbol` the end of the chain?

Reply:
I'll remove `UtilizesNullableReferenceTypes` in a follow-up PR. It wasn't simple enough to include in this PR.

---
In reply to: [196960121](https://github.com/dotnet/roslyn/pull/27807#discussion_r196960121) [](ancestors = 196960121)